jmclient/jmclient/taker.py

```diff
@@ -40,8 +41,9 @@ class _MakerTxData(NamedTuple):
         change_amount: Any
         real_cjfee: Any
         utxo_list: Any = None
-        cj_addr: Any = None
-        change_addr: Any = None
+        cj_addr: Optional[str] = None
```

Comment:
Am I wrong, or are these changes not needed? The address info in the `_MakerTxData` object are the change and coinjoin addresses of the *maker* - I don't think we're labelling these are we?

Reply:
Yes, I think you are right.

Reply:
But will keep typehint change from `Any` to `Optional[str]`.